### PR TITLE
Merge release 97.4

### PR DESCRIPTION
**Pull request by bendk**
Merging the release-v97.4 branch back to main.

### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - Note:
    - For changes that need extra cross-platform testing, consider adding `[ci full]` to the PR title.
    - If this pull request includes a breaking change, consider [cutting a new release](https://github.com/mozilla/application-services/blob/main/docs/howtos/cut-a-new-release.md) after merging.
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry in [CHANGES_UNRELEASED.md](../CHANGES_UNRELEASED.md) or an explanation of why it does not need one
  - Any breaking changes to Swift or Kotlin binding APIs are noted explicitly
- [ ] **Dependencies**: This PR follows our [dependency management guidelines](https://github.com/mozilla/application-services/blob/main/docs/dependency-management.md)
  - Any new dependencies are accompanied by a summary of the due dilligence applied in selecting them.

[Branch builds](https://github.com/mozilla/application-services/blob/main/docs/howtos/branch-builds.md): add `[firefox-android: branch-name]` to the PR title.

### CODE REVIEW

**Comment by tarikeshaq** [comment]
We can probably merge this PR with the upcoming one for 97.4.1, but LGTM

**Review by tarikeshaq — APPROVED** [review]

**Comment by bendk** [comment]
> We can probably merge this PR with the upcoming one for 97.4.1, but LGTM

Yeah, that makes sense.  I just pushed a couple more commits, does this PR still look good?

File: components/support/sql/src/conn_ext.rs [hunk 1]
```diff
@@ -76,6 +76,14 @@ pub trait ConnExt {
         Ok(res)
     }
 
+    /// Return true if a query returns any rows
+    fn exists<P: Params>(&self, sql: &str, params: P) -> SqlResult<bool> {
```
Comment (tarikeshaq) [inline components/support/sql/src/conn_ext.rs:80]: is GitHub confused or did this not land to `main` already?

**Review by tarikeshaq — APPROVED** [review]
A little confused why the Rust code is shown here, but I'm assuming the PR to land the code also landed in the release branch? regardless looks good to me

**Comment by bendk** [comment]
I think my merge got messed up, just pushed a new one.

**Review by tarikeshaq — APPROVED** [review]
:shipit:

**Pull request merged**